_opensearch/rest-api/cluster-health.md

```diff
@@ -59,6 +60,7 @@ wait_for_status | Enum | Wait until the cluster is in a specific state or better
   "timed_out" : false,
   "number_of_nodes" : 2,
   "number_of_data_nodes" : 2,
+  "discovered_master" : true,
```

Comment:
BTW, maybe we need to replace this with `discovered_cluster_manager` for 2.x doc release?
See https://github.com/opensearch-project/OpenSearch/pull/2437

Reply:
Should be covered by our 2.0.0 changes.